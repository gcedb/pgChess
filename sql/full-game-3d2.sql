TRUNCATE games;

--
-- Full game, 3 moves, depth 2
--

CALL load_game('rnbqkbnr/ppp1pppp/8/3p4/4P3/8/PPPP1PPP/RNBQKBNR w QKqk - 0 1');
CALL ui_loop(iter := 3, depth_target := 2, time_target := NULL , regress := true);
