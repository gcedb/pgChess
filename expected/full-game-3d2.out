psql:test/sql/regress/_.sql:51: INFO:  3 turns to go (1 games)
ID: rnbqkbnr/ppp1pppp/8/3p4/4P3/8/PPPP1PPP/RNBQKBNR w KQkq - 0 1
psql:test/sql/regress/_.sql:51: INFO:  Score: -3.100
Choices:
	0.300 {2.100,2.600,1.400,1.800,1.800} rnbqkbnr/ppp1pppp/8/1B1p4/4P3/8/PPPP1PPP/RNBQK1NR b KQkq - 1 1
	-0.100 {0.300,-0.100,1.200,1.100,0.100,0.200,0.200,1.100,0.400,0.400,-0.100,-0.100,0.100,0.100,0.100,1.500,-0.100,0.600,0.400,0.500,0.200,0.000,-0.500,0.100,0.000,-0.200} rnbqkbnr/ppp1pppp/8/3p3Q/4P3/8/PPPP1PPP/RNB1KBNR b KQkq - 1 1
	-0.600 {0.300,0.200,1.500,0.500,0.100,0.200,0.200,1.000,0.300,-0.100,0.100,0.100,-0.100,-0.100,0.100,0.100,0.100,1.500,-0.100,0.600,0.300,0.000,0.100,0.000,-0.100,-0.500,0.100,0.000,-0.400} rnbqkbnr/ppp1pppp/8/3p4/4P3/5Q2/PPPP1PPP/RNB1KBNR b KQkq - 1 1
	-0.700 {0.300,1.300,0.800,0.100,0.200,0.200,1.000,0.100,-0.300,0.200,0.100,-0.100,-0.100,0.100,0.100,0.100,1.500,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,-0.500,0.100,0.000,-0.400} rnbqkbnr/ppp1pppp/8/3p4/3PP3/8/PPP2PPP/RNBQKBNR b KQkq - 0 1
	-1.000 {0.300,0.200,0.800,0.100,0.200,0.200,0.400,-0.300,0.100,0.100,-0.100,-0.100,0.100,0.100,0.100,1.100,0.100,0.700,0.100,-0.100,0.100,0.000,-0.100,-0.500,0.100,0.000,-0.500} rnbqkbnr/ppp1pppp/8/3pP3/8/8/PPPP1PPP/RNBQKBNR b KQkq - 0 1
	-1.200 {0.300,0.100,1.400,0.800,0.100,0.100,0.200,1.000,0.100,-0.300,0.200,0.100,-0.100,-0.100,0.100,0.100,0.100,1.500,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,-0.500,0.100,0.000,-0.400} rnbqkbnr/ppp1pppp/8/3p4/4P3/3P4/PPP2PPP/RNBQKBNR b KQkq - 0 1
	-1.300 {0.300,0.300,1.400,0.800,0.100,0.200,0.200,1.000,0.100,-0.300,0.100,0.100,-0.100,-0.100,0.100,0.100,0.100,1.500,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,-0.500,0.100,0.000,-0.400} rnbqkbnr/ppp1pppp/8/3p4/4P3/2N5/PPPP1PPP/R1BQKBNR b KQkq - 1 1
	-1.400 {0.300,0.200,1.500,0.800,0.100,0.200,0.200,0.900,0.100,-0.300,0.100,0.100,-0.100,-0.100,0.100,0.100,0.100,1.400,-0.100,0.500,0.200,-0.100,0.100,0.000,-0.100,-0.500,0.100,0.000,-0.400} rnbqkbnr/ppp1pppp/8/3p4/1P2P3/8/P1PP1PPP/RNBQKBNR b KQkq - 0 1
	-1.400 {0.500,0.200,1.500,0.800,0.100,0.200,0.200,1.000,0.100,-0.300,0.100,0.100,-0.100,-0.100,0.100,0.100,0.100,1.400,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,-0.500,0.100,0.000,-0.400} rnbqkbnr/ppp1pppp/8/3p4/4P3/6P1/PPPP1P1P/RNBQKBNR b KQkq - 0 1
	-1.400 {0.100,0.200,1.500,0.800,0.100,0.200,0.200,1.000,0.100,-0.300,0.100,0.100,-0.100,-0.100,0.100,0.100,0.100,1.500,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,-0.500,0.100,0.000,-0.400} rnbqkbnr/ppp1pppp/8/3p4/4P2P/8/PPPP1PP1/RNBQKBNR b KQkq - 0 1
	-1.400 {0.300,0.200,1.500,0.800,0.100,0.200,0.200,1.000,0.100,-0.300,0.100,0.100,-0.100,-0.100,0.100,0.100,0.100,1.500,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,-0.500,0.100,0.000,-0.400} rnbqkbnr/ppp1pppp/8/3p4/4P3/1P6/P1PP1PPP/RNBQKBNR b KQkq - 0 1
	-1.400 {0.300,0.200,1.500,0.800,0.100,0.200,0.200,1.000,0.100,-0.300,0.100,0.100,-0.100,-0.100,0.100,0.100,0.100,1.500,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,-0.500,0.100,0.000,-0.400} rnbqkbnr/ppp1pppp/8/3p4/4P3/2P5/PP1P1PPP/RNBQKBNR b KQkq - 0 1
	-1.400 {0.300,0.200,1.500,0.800,0.100,0.200,0.200,1.000,0.100,-0.300,0.100,0.100,-0.100,-0.100,0.100,0.100,0.100,1.500,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,-0.500,0.100,0.000,-0.400} rnbqkbnr/ppp1pppp/8/3p4/P3P3/8/1PPP1PPP/RNBQKBNR b KQkq - 0 1
	-1.500 {0.300,0.200,1.500,0.800,0.100,0.200,0.200,1.000,0.100,-0.300,0.100,0.100,-0.100,-0.100,0.100,0.100,0.100,1.500,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,-0.500,0.100,0.000,-0.400} rnbqkbnr/ppp1pppp/8/3p4/4P3/N7/PPPP1PPP/R1BQKBNR b KQkq - 1 1
	-1.500 {0.300,0.200,1.500,0.800,0.100,0.200,0.200,1.000,0.200,-0.300,0.100,0.100,-0.100,-0.100,0.100,0.100,0.100,1.300,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,-0.500,0.100,0.000,-0.400} rnbqkbnr/ppp1pppp/8/3p4/4PP2/8/PPPP2PP/RNBQKBNR b KQkq - 0 1
	-1.600 {0.300,1.400,0.200,1.500,0.800,0.100,0.100,0.200,1.000,0.100,-0.300,0.100,0.100,-0.100,-0.100,0.100,0.100,0.100,1.500,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,-0.500,0.100,0.000,-0.400} rnbqkbnr/ppp1pppp/8/3p4/2P1P3/8/PP1P1PPP/RNBQKBNR b KQkq - 0 1
	-1.600 {0.300,0.000,1.400,0.800,0.100,0.200,0.200,1.000,0.100,-0.300,0.100,0.100,-0.100,-0.100,0.100,0.100,0.100,1.500,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,-0.500,0.100,0.000,-0.400} rnbqkbnr/ppp1pppp/8/3p4/4P3/3B4/PPPP1PPP/RNBQK1NR b KQkq - 1 1
	-1.600 {1.000,0.200,1.500,0.700,0.100,0.200,0.200,1.000,0.100,-0.300,0.100,0.100,-0.100,-0.100,0.100,0.100,0.100,1.500,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,-0.500,0.100,0.000,-0.400} rnbqkbnr/ppp1pppp/8/3p4/4P3/7P/PPPP1PP1/RNBQKBNR b KQkq - 0 1
	-1.600 {0.300,0.200,1.400,0.600,0.100,0.200,0.200,1.000,0.100,-0.300,0.100,0.100,-0.100,-0.100,0.100,0.100,0.100,1.500,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,-0.500,0.100,0.000,-0.400} rnbqkbnr/ppp1pppp/8/3p4/4P3/8/PPPPQPPP/RNB1KBNR b KQkq - 1 1
	-1.600 {0.300,0.200,1.500,0.800,0.100,0.200,0.200,1.000,0.100,-0.300,0.100,0.100,-0.100,-0.100,0.100,0.100,0.100,1.500,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,-0.500,0.100,0.000,-0.400} rnbqkbnr/ppp1pppp/8/3p4/4P3/P7/1PPP1PPP/RNBQKBNR b KQkq - 0 1
	-1.700 {0.200,1.500,1.700,0.100,0.200,0.200,1.000,0.000,-0.200,0.100,0.100,-0.100,-0.100,0.100,0.100,0.100,1.500,-0.100,0.700,0.200,-0.100,0.100,0.000,-0.100,-0.400,0.100,0.000,-0.300} rnbqkbnr/ppp1pppp/8/3p4/4P1P1/8/PPPP1P1P/RNBQKBNR b KQkq - 0 1
	-1.700 {0.300,0.200,1.500,0.600,0.100,0.200,0.200,1.000,0.100,-0.300,0.100,0.100,-0.100,-0.100,0.100,0.100,0.100,1.500,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,-0.500,0.100,0.000,-0.400} rnbqkbnr/ppp1pppp/8/3p4/4P3/8/PPPPBPPP/RNBQK1NR b KQkq - 1 1
	-1.800 {0.300,0.200,1.600,0.400,0.100,0.200,0.200,1.000,0.100,-0.300,0.100,0.100,-0.100,-0.100,0.100,0.100,0.100,1.500,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,-0.500,0.100,0.000,-0.400} rnbqkbnr/ppp1pppp/8/3p4/4P3/5N2/PPPP1PPP/RNBQKB1R b KQkq - 1 1
	-1.900 {0.300,0.200,1.500,0.300,0.100,0.200,0.200,1.000,0.100,-0.300,0.100,0.100,-0.100,-0.100,0.100,0.100,0.100,1.500,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,-0.500,0.100,0.000,-0.400} rnbqkbnr/ppp1pppp/8/3p4/4P3/5P2/PPPP2PP/RNBQKBNR b KQkq - 0 1
	-2.000 {0.300,0.800,0.100,0.200,0.200,3.000,1.000,0.400,-0.300,0.100,0.100,-0.100,-0.100,0.100,0.000,0.100,1.600,-0.100,0.600,0.300,-0.100,0.100,0.000,-0.100,-0.600,0.100,-0.100,-0.700} rnbqkbnr/ppp1pppp/8/3P4/8/8/PPPP1PPP/RNBQKBNR b KQkq - 0 1
	-2.200 {0.300,0.200,1.500,0.500,0.100,0.100,0.200,1.000,0.100,-0.300,0.100,0.100,-0.100,-0.100,0.100,0.100,0.100,1.500,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,-0.500,0.100,0.000,-0.400} rnbqkbnr/ppp1pppp/8/3p4/4P3/8/PPPPNPPP/RNBQKB1R b KQkq - 1 1
	-3.000 {0.300,0.300,1.700,2.400,0.100,0.100,0.200,1.000,0.100,-0.300,0.100,0.100,-0.100,-0.100,0.100,0.100,0.100,1.500,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,-0.500,0.100,0.000,-0.400} rnbqkbnr/ppp1pppp/8/3p4/4P3/8/PPPPKPPP/RNBQ1BNR b KQkq - 1 1
	-3.300 {0.300,0.200,1.500,0.800,0.300,0.200,1.000,0.100,-0.300,0.100,0.100,3.600,3.400,-0.100,0.100,0.100,1.500,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,-0.500,0.100,0.000,-0.400} rnbqkbnr/ppp1pppp/B7/3p4/4P3/8/PPPP1PPP/RNBQK1NR b KQkq - 1 1
	-3.500 {3.400,0.200,1.500,0.800,0.100,0.200,0.200,1.000,0.100,-0.300,0.100,0.100,-0.100,-0.100,0.100,0.100,0.100,1.500,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,-0.500,0.100,0.000,-0.400} rnbqkbnr/ppp1pppp/8/3p4/4P3/7N/PPPP1PPP/RNBQKB1R b KQkq - 1 1
	-3.900 {0.300,4.000,-0.100,1.200,0.800,0.100,0.300,0.100,1.000,0.100,-0.300,0.100,0.100,-0.100,-0.100,0.100,0.100,0.100,1.500,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,-0.500,0.100,0.000,-0.400} rnbqkbnr/ppp1pppp/8/3p4/2B1P3/8/PPPP1PPP/RNBQK1NR b KQkq - 1 1
	-10.100 {0.200,1.400,11.400,0.100,0.200,0.200,1.000,0.500,0.300,0.200,0.200,-0.100,-0.100,0.100,0.100,0.100,1.500,0.200,1.000,0.200,-0.100,0.200,0.000,-0.100,-0.200,0.300,0.200} rnbqkbnr/ppp1pppp/8/3p4/4P1Q1/8/PPPP1PPP/RNB1KBNR b KQkq - 1 1
psql:test/sql/regress/_.sql:51: INFO:  2 turns to go (898 games)
ID: rnbqkbnr/ppp1pppp/8/1B1p4/4P3/8/PPPP1PPP/RNBQK1NR b KQkq - 1 1
psql:test/sql/regress/_.sql:51: INFO:  Score: 0.500
Choices:
	-0.700 {-0.400,0.000,-0.300,0.100,-0.600,-0.200,-0.100,-0.100,0.100,0.100,0.100,-0.500,0.400,-0.100,0.900,-0.400,-0.100,0.000,-0.100,-0.500,0.000,0.000,-0.300,-0.200,0.800,0.000,1.300,-0.200,0.100,1.100,0.100,1.400,-0.100,3.300} rnbqkbnr/pp2pppp/2p5/1B1p4/4P3/8/PPPP1PPP/RNBQK1NR w KQkq - 0 2
	-2.900 {-0.900,0.000,-0.300,0.100,-1.100,-0.200,-0.100,-0.100,0.100,0.100,0.100,-1.000,0.400,-0.100,0.900,-0.400,-0.100,0.000,-0.100,-0.600,0.000,0.000,-0.800,-0.200,0.800,0.000,1.300,-0.200,0.100,1.100,0.100,1.400,-0.600,5.000} r1bqkbnr/ppp1pppp/2n5/1B1p4/4P3/8/PPPP1PPP/RNBQK1NR w KQkq - 2 2
	-3.200 {-1.000,0.000,-0.300,0.100,-1.200,-0.200,-0.100,-0.100,0.100,0.100,0.100,-1.100,0.400,-0.100,0.900,-0.400,-0.100,0.000,-0.100,-0.600,0.000,0.000,-0.900,-0.200,0.800,0.000,1.000,-0.300,0.100,1.100,0.100,1.400,-0.700,-0.300,4.600} r1bqkbnr/pppnpppp/8/1B1p4/4P3/8/PPPP1PPP/RNBQK1NR w KQkq - 2 2
	-3.200 {-1.100,0.000,-0.300,0.100,-1.300,-0.200,-0.100,-0.100,0.100,0.100,0.100,-1.200,0.400,-0.100,0.900,-0.400,-0.100,0.000,-0.100,-0.700,0.000,0.000,-1.000,-0.200,0.800,0.000,1.000,-0.300,0.100,1.100,0.100,1.400,-0.800,-0.200,5.000} rn1qkbnr/pppbpppp/8/1B1p4/4P3/8/PPPP1PPP/RNBQK1NR w KQkq - 2 2
	-9.100 {-1.200,0.000,-0.300,0.100,-1.400,-0.200,-0.100,-0.100,0.100,0.100,0.100,-1.300,0.400,-0.100,0.900,-0.400,-0.100,0.000,-0.100,-0.700,0.000,0.000,-1.100,-0.200,0.800,0.000,1.000,-0.300,0.100,1.100,0.100,1.400,-0.900,-0.200,10.900} rnb1kbnr/pppqpppp/8/1B1p4/4P3/8/PPPP1PPP/RNBQK1NR w KQkq - 2 2
psql:test/sql/regress/_.sql:51: INFO:  1 turns to go (1071 games)
ID: rnbqkbnr/pp2pppp/2p5/1B1p4/4P3/8/PPPP1PPP/RNBQK1NR w KQkq - 0 2
psql:test/sql/regress/_.sql:51: INFO:  Score: -0.100
Choices:
	-1.900 {0.300,0.000,1.400,0.800,0.700,0.000,0.200,0.100,1.000,0.100,-0.300,0.100,0.100,0.100,-0.100,0.600,-0.100,1.000,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,0.500,-0.400,-0.400,-0.300,-0.400} rnbqkbnr/pp2pppp/2p5/3p4/4P3/3B4/PPPP1PPP/RNBQK1NR b KQkq - 1 2
	-1.900 {0.300,0.200,1.500,0.800,0.900,0.000,0.200,0.100,1.000,0.100,-0.300,0.100,0.100,0.100,-0.100,0.600,-0.100,1.000,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,0.500,-0.400,-0.400,-0.300,-0.400} rnbqkbnr/pp2pppp/2p5/3p4/4P3/8/PPPP1PPP/RNBQKBNR b KQkq - 1 2
	-2.000 {0.300,0.200,1.500,0.800,0.700,-0.100,0.400,1.000,0.100,-0.300,0.100,0.100,0.100,-0.100,0.600,-0.100,1.000,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,0.500,-0.300,-0.300,-0.200,-0.400} rnbqkbnr/pp2pppp/2p5/3p4/B3P3/8/PPPP1PPP/RNBQK1NR b KQkq - 1 2
	-2.100 {0.300,0.200,1.500,0.600,0.900,0.000,0.200,0.100,1.000,0.100,-0.300,0.100,0.100,0.100,-0.100,0.600,-0.100,1.000,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,0.500,-0.400,-0.400,-0.300,-0.400} rnbqkbnr/pp2pppp/2p5/3p4/4P3/8/PPPPBPPP/RNBQK1NR b KQkq - 1 2
	-2.400 {0.300,0.100,1.400,1.100,0.800,0.000,3.800,1.100,0.400,0.400,0.100,0.000,0.400,-0.100,1.000,-0.100,0.600,0.400,0.500,0.200,0.000,0.500,-0.300,-0.300,-0.200,-0.200} rnbqkbnr/pp2pppp/2p5/1B1p3Q/4P3/8/PPPP1PPP/RNB1K1NR b KQkq - 1 2
	-2.800 {0.300,0.800,0.700,0.000,3.900,2.600,1.000,0.400,-0.300,0.100,0.100,0.100,0.000,0.300,-0.100,1.100,-0.100,0.600,0.300,-0.100,0.100,0.000,-0.100,0.400,-0.300,-0.200,-0.200,-0.600} rnbqkbnr/pp2pppp/2p5/1B1P4/8/8/PPPP1PPP/RNBQK1NR b KQkq - 0 2
	-2.900 {0.300,0.200,1.500,0.500,0.800,0.000,3.800,1.000,0.300,-0.100,0.100,0.100,0.100,0.000,0.400,-0.100,1.000,-0.100,0.600,0.300,0.000,0.100,0.000,-0.100,0.500,-0.300,-0.300,-0.200,-0.400} rnbqkbnr/pp2pppp/2p5/1B1p4/4P3/5Q2/PPPP1PPP/RNB1K1NR b KQkq - 1 2
	-3.100 {6.400,6.100,1.700,1.700,2.000} rnbqkbnr/pp2pppp/2B5/3p4/4P3/8/PPPP1PPP/RNBQK1NR b KQkq - 0 2
	-3.200 {0.300,1.300,0.800,4.000,0.000,3.800,1.000,0.100,-0.300,0.200,0.100,0.100,0.000,0.200,-0.100,1.000,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,0.500,-0.300,-0.300,-0.200,-0.400} rnbqkbnr/pp2pppp/2p5/1B1p4/3PP3/8/PPP2PPP/RNBQK1NR b KQkq - 0 2
	-3.300 {0.300,0.100,1.400,0.800,3.700,0.000,3.500,1.000,0.100,-0.300,0.200,0.100,0.100,0.000,0.400,-0.100,1.000,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,0.500,-0.300,-0.300,-0.200,-0.400} rnbqkbnr/pp2pppp/2p5/1B1p4/4P3/3P4/PPP2PPP/RNBQK1NR b KQkq - 0 2
	-3.400 {0.300,0.200,1.400,0.600,0.800,0.000,3.500,1.000,0.100,-0.300,0.100,0.100,0.100,0.000,0.400,-0.100,1.000,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,0.500,-0.300,-0.300,-0.200,-0.400} rnbqkbnr/pp2pppp/2p5/1B1p4/4P3/8/PPPPQPPP/RNB1K1NR b KQkq - 1 2
	-3.600 {0.300,0.300,1.400,0.800,0.500,0.000,3.700,1.000,0.100,-0.300,0.100,0.100,0.100,0.000,0.400,-0.100,1.000,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,0.500,-0.300,-0.300,-0.200,-0.400} rnbqkbnr/pp2pppp/2p5/1B1p4/4P3/2N5/PPPP1PPP/R1BQK1NR b KQkq - 1 2
	-3.600 {0.300,1.400,0.200,1.500,0.800,0.800,0.000,3.400,1.000,0.100,-0.300,0.100,0.100,0.100,0.000,0.400,-0.100,1.000,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,0.500,-0.300,-0.300,-0.200,-0.400} rnbqkbnr/pp2pppp/2p5/1B1p4/2P1P3/8/PP1P1PPP/RNBQK1NR b KQkq - 0 2
	-3.700 {0.300,0.200,1.500,0.800,0.300,0.000,3.700,0.900,0.100,-0.300,0.100,0.100,0.100,0.000,0.400,-0.100,0.900,-0.100,0.500,0.200,-0.100,0.100,0.000,-0.100,0.500,-0.300,-0.300,-0.200,-0.400} rnbqkbnr/pp2pppp/2p5/1B1p4/1P2P3/8/P1PP1PPP/RNBQK1NR b KQkq - 0 2
	-3.700 {0.100,0.200,1.500,0.800,0.800,0.000,3.800,1.000,0.100,-0.300,0.100,0.100,0.100,0.000,0.400,-0.100,1.000,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,0.500,-0.300,-0.300,-0.200,-0.400} rnbqkbnr/pp2pppp/2p5/1B1p4/4P2P/8/PPPP1PP1/RNBQK1NR b KQkq - 0 2
	-3.700 {0.300,0.200,1.500,0.800,0.800,0.000,3.800,1.000,0.100,-0.300,0.100,0.100,0.100,0.000,0.400,-0.100,1.000,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,0.500,-0.300,-0.300,-0.200,-0.400} rnbqkbnr/pp2pppp/2p5/1B1p4/4P3/1P6/P1PP1PPP/RNBQK1NR b KQkq - 0 2
	-3.700 {0.300,0.200,1.500,0.800,0.500,0.000,3.800,1.000,0.100,-0.300,0.100,0.100,0.100,0.000,0.400,-0.100,1.000,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,0.500,-0.300,-0.300,-0.200,-0.400} rnbqkbnr/pp2pppp/2p5/1B1p4/4P3/2P5/PP1P1PPP/RNBQK1NR b KQkq - 0 2
	-3.700 {3.300,0.100,1.400,0.700,0.700,-0.100,3.700,0.900,0.000,-0.400,0.000,0.000,0.000,-0.100,0.300,-0.200,0.900,-0.200,0.500,0.100,-0.200,0.000,-0.100,-0.200,0.400,-0.400,-0.400,-0.300,-0.500} rnbqkbnr/pp2pppp/2p5/1B1p4/4P3/7N/PPPP1PPP/RNBQK2R b KQkq - 1 2
	-3.700 {0.500,0.200,1.500,0.800,0.600,0.000,3.700,1.000,0.100,-0.300,0.100,0.100,0.100,0.000,0.400,-0.100,1.000,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,0.500,-0.300,-0.300,-0.200,-0.400} rnbqkbnr/pp2pppp/2p5/1B1p4/4P3/8/PPPP1PPP/RNBQ1KNR b KQkq - 1 2
	-3.700 {0.300,0.200,1.500,0.800,0.700,0.000,3.700,1.000,0.100,-0.300,0.100,0.100,0.100,0.000,0.400,-0.100,1.000,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,0.500,-0.300,-0.300,-0.200,-0.400} rnbqkbnr/pp2pppp/2p5/1B1p4/P3P3/8/1PPP1PPP/RNBQK1NR b KQkq - 0 2
	-3.700 {0.300,0.200,0.800,0.800,0.000,3.800,0.400,-0.300,0.100,0.100,0.100,0.000,0.400,-0.100,0.600,0.100,0.700,0.100,-0.100,0.100,0.000,-0.100,0.200,-0.300,-0.300,-0.200,-0.500} rnbqkbnr/pp2pppp/2p5/1B1pP3/8/8/PPPP1PPP/RNBQK1NR b KQkq - 0 2
	-3.800 {0.200,0.100,1.500,0.300,0.700,-0.100,3.700,0.900,0.000,-0.400,0.000,0.000,0.000,-0.100,0.300,-0.200,0.900,-0.200,0.500,0.100,-0.200,0.000,-0.100,-0.200,0.400,-0.400,-0.400,-0.300,-0.500} rnbqkbnr/pp2pppp/2p5/1B1p4/4P3/5N2/PPPP1PPP/RNBQK2R b KQkq - 1 2
	-3.800 {0.300,0.300,1.700,3.000,0.600,0.000,3.600,1.000,0.100,-0.300,0.100,0.100,0.100,0.000,0.500,-0.100,1.000,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,0.500,-0.300,-0.300,-0.200,-0.400} rnbqkbnr/pp2pppp/2p5/1B1p4/4P3/8/PPPPKPPP/RNBQ2NR b KQkq - 1 2
	-3.800 {0.200,0.100,1.400,0.400,0.700,-0.100,3.500,0.900,0.000,-0.400,0.000,0.000,0.000,-0.100,0.300,-0.200,0.900,-0.200,0.500,0.100,-0.200,0.000,-0.100,-0.200,0.400,-0.400,-0.400,-0.300,-0.500} rnbqkbnr/pp2pppp/2p5/1B1p4/4P3/8/PPPPNPPP/RNBQK2R b KQkq - 1 2
	-3.800 {0.300,0.200,1.500,0.800,0.700,0.000,3.700,1.000,0.100,-0.300,0.100,0.100,0.100,0.000,0.400,-0.100,1.000,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,0.500,-0.300,-0.300,-0.200,-0.400} rnbqkbnr/pp2pppp/2p5/1B1p4/4P3/N7/PPPP1PPP/R1BQK1NR b KQkq - 1 2
	-3.800 {0.300,0.200,1.500,0.800,0.800,0.000,3.800,1.000,0.200,-0.300,0.100,0.100,0.100,0.000,0.600,-0.100,0.800,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,0.300,-0.300,-0.300,-0.200,-0.400} rnbqkbnr/pp2pppp/2p5/1B1p4/4PP2/8/PPPP2PP/RNBQK1NR b KQkq - 0 2
	-3.900 {0.600,0.200,1.500,0.800,0.800,0.000,3.800,1.000,0.100,-0.300,0.100,0.100,0.100,0.000,0.400,-0.100,0.900,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,0.400,-0.300,-0.300,-0.200,-0.400} rnbqkbnr/pp2pppp/2p5/1B1p4/4P3/6P1/PPPP1P1P/RNBQK1NR b KQkq - 0 2
	-3.900 {1.000,0.200,1.500,0.700,0.800,0.000,3.800,1.000,0.100,-0.300,0.100,0.100,0.100,0.000,0.400,-0.100,1.000,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,0.500,-0.300,-0.300,-0.200,-0.400} rnbqkbnr/pp2pppp/2p5/1B1p4/4P3/7P/PPPP1PP1/RNBQK1NR b KQkq - 0 2
	-3.900 {0.300,0.200,1.500,0.800,0.700,0.000,3.800,1.000,0.100,-0.300,0.100,0.100,0.100,0.000,0.400,-0.100,1.000,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,0.500,-0.300,-0.300,-0.200,-0.400} rnbqkbnr/pp2pppp/2p5/1B1p4/4P3/P7/1PPP1PPP/RNBQK1NR b KQkq - 0 2
	-3.900 {0.300,0.200,1.500,0.800,1.000,0.300,0.100,1.000,0.100,-0.300,0.100,0.100,3.800,3.400,0.600,-0.300,1.000,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,0.500,-0.400,-0.400,-0.300,-0.400} rnbqkbnr/pp2pppp/B1p5/3p4/4P3/8/PPPP1PPP/RNBQK1NR b KQkq - 1 2
	-4.000 {0.200,1.500,1.700,0.800,0.000,3.800,1.000,0.000,-0.200,0.100,0.100,0.100,0.000,0.400,-0.100,1.000,-0.100,0.700,0.200,-0.100,0.100,0.000,-0.100,0.500,-0.200,-0.300,-0.200,-0.300} rnbqkbnr/pp2pppp/2p5/1B1p4/4P1P1/8/PPPP1P1P/RNBQK1NR b KQkq - 0 2
	-4.200 {0.300,0.200,1.500,0.300,0.800,0.000,3.800,1.000,0.100,-0.300,0.100,0.100,0.100,0.000,0.600,-0.100,1.000,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,0.500,-0.300,-0.300,-0.200,-0.400} rnbqkbnr/pp2pppp/2p5/1B1p4/4P3/5P2/PPPP2PP/RNBQK1NR b KQkq - 0 2
	-4.300 {0.300,4.000,-0.100,1.200,0.800,0.900,0.000,0.300,0.000,1.000,0.100,-0.300,0.100,0.100,0.100,-0.100,0.600,-0.100,1.000,-0.100,0.600,0.200,-0.100,0.100,0.000,-0.100,0.500,-0.400,-0.400,-0.300,-0.400} rnbqkbnr/pp2pppp/2p5/3p4/2B1P3/8/PPPP1PPP/RNBQK1NR b KQkq - 1 2
	-10.100 {0.200,1.400,11.400,0.800,0.000,3.800,1.000,0.500,0.300,0.200,0.200,0.100,0.000,0.400,-0.100,1.000,0.200,1.000,0.200,-0.100,0.200,0.000,-0.100,0.500,0.000,-0.100,0.000} rnbqkbnr/pp2pppp/2p5/1B1p4/4P1Q1/8/PPPP1PPP/RNB1K1NR b KQkq - 1 2
psql:test/sql/regress/_.sql:51: INFO:  0 turns to go (2029 games)
ID: rnbqkbnr/pp2pppp/2p5/3p4/4P3/8/PPPP1PPP/RNBQKBNR b KQkq - 1 2
psql:test/sql/regress/_.sql:51: INFO:  game ended after 3 iterations
